basin3d/core/schema/query.py

```diff
@@ -95,21 +105,29 @@ def __init__(self, **data):
                 data[field] = isinstance(data[field], str) and data[field].upper() or data[field]
         super().__init__(**data)
 
+    def get_prefixed_fields(self) -> list:
+        return ['monitoring_feature', 'parent_feature']
+
```

Comment:
This should be a field that is a pydantic ClassVar instead

```suggestion
   prefixed_fields: ClassVar[List[str]] = ['monitoring_feature', 'parent_feature']

```

Reply:
OK. Done.